ApollosProject/apollos-prototype
# Adds algorithm to ActionListCard to view content from the sermon throughout the week.

## DESCRIPTION

### What does this PR do, or why is it needed?

This PR adds an algorithm to the ActionListCard so that you can see content related to the sermon throughout the week. 

This Algorithm works by showing all children of a sermon that are published. A church could attach articles to a sermon, and then throughout the week those articles could go live leading to new content appearing on the home feed daily. 

### What design trade-offs/decisions were made?

Chose to integrate content within the persona feed. 

### How do I test this PR?

1  Open the app and login. 
 - You should see the "Live for Freedom Toolkit" alongside the persona items in the "Explore what god calls you to today" action list. I added the "Live for Freedom Toolkit" as a child to the most recent sermon in Rock.

### What automated tests did you write?
Tests over new datasource methods.

## TODO:

- [x] Closes #874 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
